app/scripts/lib/metrics.js

```diff
@@ -135,6 +140,29 @@ define([
       return this._send(filteredData, url, false);
     },
 
+    /**
+     * Set the unique assigned random UUID for ab testing purposes
+     *
+     */
+    setUniqueId: function () {
+      if (this._metricsStorage && ! this.getUniqueId()) {
+        this._metricsStorage.set('uuid', uuid.v4());
```

Comment:
It will be. We can land this patch first, then add the `uuid` to the ResumeToken in https://github.com/mozilla/fxa-content-server/pull/2567 or the follow on to that needed for #2492 or #2563